payments/invoice/after-payment.md

```diff
@@ -25,63 +25,28 @@ sidebar:
 
 ## Options after posting a payment
 
-* **Abort:** It is possible to abort the process,
-    if the payment has no successful transactions.
-    [See the PATCH payment description][payment-description].
-* You must always follow up an Invoice Authorization with a
-    Capture or Cancel request.
-* For reversals, you will need to implement the Reversal request.
-* **If CallbackURL is set:** Whenever changes to the payment occur a
-    [Callback request][callback-request] will be posted to the callbackUrl,
-    which was generated when the payment was created.
+When you detect that the payer has reached your `completeUrl`, you need to do a
+GET request on the payment resource, which contains the paymentID generated in
+the first step, to receive the state of the transaction. You will also be able
+to see the available operations after posting a payment.
+
+{% include payment-resource.md payment-instrument="invoice" %}
+
+* **Abort:** It is possible to abort the process if the payment has no
+  successful transactions. [See the PATCH payment
```

Comment:
Please try to use the logical names of the operations, not the HTTP methods. So instead of `PATCH`, use the `rel` value or simply write `Abort`. The HTTP method is an implementation detail, the operation name is not.